mne/io/meas_info.py

```diff
@@ -150,13 +150,18 @@ class MontageMixin(object):
     """Mixin for Montage setting."""
 
     @verbose
-    def set_montage(self, montage, match_case=True, verbose=None):
+    def set_montage(self, montage, match_case=True,
+                    on_missing='raise', verbose=None):
         """Set EEG sensor configuration and head digitization.
 
         Parameters
         ----------
         %(montage)s
         %(match_case)s
+        on_missing : str
+            Either 'raise' to raise an error when
+            channels have missing coordinates,
+            or 'ignore' to set channels to np.nan and set montage.
```

Comment:
This can be moved to `mne/utils/docs.py` and here you can just do `(on_missing_montage)s`, but it would be good to add:

```suggestion
            or 'ignore' to set channels to np.nan and set montage.

            .. versionadded:: 0.20.1
```

because we will want to backport this